js-miniapp-bridge/src/types/points.ts

```diff
@@ -0,0 +1,6 @@
+/** Points type. */
+export interface Points {
```

Comment:
You also need to import this type to `js-miniapp-sdk/src/index.html` and then export it from that file so that the type is publicly accessible.